README.md

```diff
@@ -1,6 +1,130 @@
 
-## Usage: 
-git clone --recursive https://github.com/Open-Network-Insight/open-network-insight.git 
+While current threat intelligence tools help, identifying unknown threats and attacks remains a challenge. Open Network Insight provides tools to accelerate companies’ ability to expose suspicious connections and previously unseen attacks using flow and packet analysis technologies. 
+<br><br>
+
+![oni logo](docs/oni-guy.png "ONI")![](docs/oni-logo.png "ONI")
+
+----
+
+## **Overview**
+![oni approach](docs/oni-approach.png "ONI Approach")
+
+With the arrival of big data platforms, security organizations can now make data-driven decisions about how they protect their assets.  Records of network traffic, captured as network flows, are often stored and analyzed for use in network management.  An organization can use this same information to gain insight into what channels corporate information flows through. 
+
+By taking into account additional context such as prevalent attacks and key protocols to the organization, the security team can develop a strategy that applies the right amount of per-channel risk mitigation based on the value of the data flowing through it.  For an organization, we call this “the port perspective”. 
+
+There are two vectors that all organizations should evaluate:
+
+ * A “wide enough, deep enough” protection strategy that involves both edge prevention and sophisticated detection of unusual behavior
+
+ * A deep inspection of key protocols using methods that can scale to the volume of data flowing across that channel
+
+While inspecting specific, unique flows of data that may be important for individual organizations, all organizations can realize significant risk reduction from analysis of network flows and DNS (domain name service) replies.
+
+**Open Network Insight**  by leveraging strong technology in both Big Data and Scientific Computing disciplines is a solution intended to support this strategy by focusing on “hard security problems” detecting events such as lateral movement, side-channel data escapes, insider issues, or stealthy behavior in general.  
+
+ 
+### **Telemetry**
+* Flows.
+* DNS (pcaps).
+
+### **Parallel Ingest Framework**
+* Open source decoders.
+* Load data in Hadoop.
+* Data transformation.
+
+
+### **Machine Learning**
+* Filter billion of events to a few thousands.
+* Unsupervised learning.
+
+### **Operational Analytics**
+* Visualization.
+* Attack heuristics.
+* Noise filter.
+
+
+## **Getting Started**
+
+ONI can be installed by following our installation manual. To get started, [check out the installation instructions in the documentation](https://github.com/Open-Network-Insight/open-network-insight/wiki).
+
+
+## If you want to get all the ONI code at one just clone it !
```

Comment:
one --> once
how about this:
## If you want all of the ONI code at once, just clone it!